src/constants/tokens.ts

```diff
@@ -155,3 +157,16 @@ export const NEAR_WHITELIST_TOKEN_ADDRESSES = [
   "v1.guild-covenant.near",
   "chill-129.meme-cooking.near",
 ]
+
+export const NEP141_STORAGE_TOKEN: BaseTokenInfo = {
```

Comment:
With this, I think we no longer need the W_NEAR_TOKEN_META constant above. Let's remove it."

Reply:
Yeah, probably. We have a lot of code to remove. We'll need to remove it at some point.